cmake/modules/SwiftSupport.cmake

```diff
@@ -9,6 +9,8 @@
 function(get_swift_host_arch result_var_name)
   if("${CMAKE_SYSTEM_PROCESSOR}" STREQUAL "x86_64")
     set("${result_var_name}" "x86_64" PARENT_SCOPE)
+  elseif("${CMAKE_SYSTEM_PROCESSOR}" STREQUAL "arm64")
```

Comment:
Probably best to match the other packages and merge the arm64 and aarch64 cases via a MATCH instead of STREQUALs.

Reply:
done.